crates/router/src/connector/nexinets/transformers.rs

```diff
@@ -100,6 +138,29 @@ pub enum NexinetsBIC {
     VanLanschot,
 }
 
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub enum NexinetsWalletDetails {
+    ApplePayToken(Box<ApplePayDetails>),
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ApplePayDetails {
+    payment_data: Option<serde_json::Value>,
```

Comment:
Why payment_data is option in applepay request?

Reply:
Yes , `payment_data` should not be an option , I have removed it and made it 'serde_json_Value' .